portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,15 +33,20 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private ArrayList<String> messages;
-
-  @Override
-  public void init(){
-    messages = new ArrayList<String>();
-  }  
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Would you mind adding a comment in a Javadoc style to explain what this function is doing? All public functions should have documentation.

http://www.drjava.org/docs/user/ch10.html

Reply:
Solved